lint-staged.config.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  "*.js": ["npm run lint:js --"],
+  "*.js": ["npm run lint:js --", "npm run format --"],
```

Comment:
was this accidental?  seems like this was something from the [previous PR](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/pull/112#discussion_r1810734443)?

Reply:
I thought so too, but `main` doesn't seem to have it. Not sure where it was lost. 

https://github.com/ProjectEvergreen/www.greenwoodjs.dev/blob/0ba63b5cac589766e4c9d67a229b733076cc85e5/lint-staged.config.js#L2

Reply:
> Hmm... it was removed. Should I revert this change then?

yes please, as it seems to work just as fine when applying the formatting just to the glob pattern.  didn't seem like it needed to be run twice.

Reply:
Ok, removed.